### DISCUSSION TITLE
Highlight "top PRs" on Profiles

### DISCUSSION THREAD

**Discussion by rattrayalex** [community/community#18673] [Other Feature Feedback, Questions, & Ideas]
Right now you have pinned repos, but it's often unclear… why.

It'd be cool to see someone's PRs that are made to repos with lots of stars, big PRs, PRs that had a lot of activity, or just PRs that the person pinned as being interesting.

**Comment by number0x0001** [comment]
Sounds neat. Maybe they will add it!
